### DISCUSSION TITLE
Multiple sonoff rf bridges - but want to sent one message to mqtt broker

### DISCUSSION THREAD

**Discussion by atv2016** [arendst/Tasmota#15885] [Q&A] [answered]
Is there a way for tasmota to send only 1 message to the mqtt broker, if there are multiple tasmota sonoff rf bridges at work receiving the same RF message? I want to use the same topic, but only send it once (to avoid messing with automations and the like).

Apparently openmqttgateway does this, but i'd rather stick with tasmota for now.

The reason for the multiple sonoff bridges is not loosing 433 messages in a large house.

**Comment by sfromis** [comment, marked as answer]
Tasmota has no coordination between bridges to filter out duplicates. One option could be a small automation getting the messages via MQTT on specific topics, doing the filtering, and then publish again to the old topic for the existing automations.

**Comment by Jason2866** [comment]
I use 2 Sonoff RF Bridges. The HomeAutomation hub subscribes both mqtt messages. Can easily processed there. Do it this way since a few years. Works well. There is no need to filter messages.
Just comparing them, and use only one of them when arrived at the same time.

Use the broker as it is. Broker is designed to have ALL infos. Just use what you need.

At a certain point a Home Automation Hub is way more easier to setup and maintain, than creating a complicated "sophisticated" setup without one.

**Comment by atv2016** [comment]
What do you mean by home automation hub? Homeassistant?
________________________________

**Reply by sfromis** [reply]
A hub is basically a smarthome server where you run central tools and automations. While Home Assistant is one tool you may be running on such a server, there are also many other tools available; I'm using Node-Red and some homebaked stuff. I could even choose to use an ESP32 with Tasmota and Berry as a small additional hub, where I can do both subscriptions, publishes, and relevant processing.
